kolibri/utils/version.py

```diff
@@ -235,7 +227,10 @@ def get_version_from_git(get_git_describe_string):
     else:
         major, minor, patch = version_split
 
-    suffix = m.group('suffix')
+    # We need to replace "-" with ".". Namely, this is done automatically in
+    # the naming of source dist .whl and .tar.gz files produced by setup.py.
+    # See: https://www.python.org/dev/peps/pep-0440/#local-version-identifiers
+    suffix = m.group('suffix').replace("-", ".")
```

Comment:
Here is the actual fix.

Reply:
I looked at the code to analyze if `suffix` will always be a regex match group, and for now, I prefer that this fails loudly, if for some insane reason there is a match in `git describe` but it fails to detect data for a suffix. So not checking for `NoneType` is intended.